tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/passes/ConfigurationPassImpl.scala

```diff
@@ -123,19 +158,17 @@ class ConfigurationPassImpl @Inject()(val options: WriteablePassOptions,
           logger.info("  > No TLC configuration found. Skipping.")
         }
         else {
-          throw new TLCConfigurationError("  > Could not find TLC config file " + basename + " given via --config option")
+          throw new TLCConfigurationError(s"  > $basename: TLC config is provided with --config, but not found")
```

Comment:
I think supplying the name of the config file that failed to load would be helpful.

Reply:
That is exactly what is kept in `basename`

Reply:
Ah I see. My mistake.